src/layouts/project/ProjectSidebar.tsx

```diff
@@ -73,30 +73,30 @@ export const ProjectSidebar = (props: ProjectSidebarProps) => {
               <NavItem 
                 tabIndex={-1}
                 className="no-hover"
-                icon={GooglePodcastsLogo}
+                icon={<RecogitoCircleLogo className="recogito-logo-circle" />}
                 label='Recogito'
```

Comment:
Can we change this to 'Projects'? I think it is more meaningful for navigation purposes.

Reply:
Agreed. I wonder if we should get rid of the logo altogether in this case, and just use a "back" caret, labeled "Projects"?